services/api_gateway/src/test/com/futurewei/alcor/apigateway/AlcorApiGatewayApplicationTest.java

```diff
@@ -0,0 +1,52 @@
+package com.futurewei.alcor.apigateway;
```

Comment:
we may also need the copy right header for test files.

Reply:
Definitely.